mama/c_cpp/src/examples/cpp/mamapublishercpp.cpp

```diff
@@ -51,7 +51,6 @@ using std::cout;
 using std::endl;
 
 static int           gCount         = 0;
-static const char *  gSource        = "";
```

Comment:
Why not just initialize this as NULL rather than an empty string and retain the ability to specify a source if requested?